.gn

```diff
@@ -31,8 +31,8 @@ default_args = {
   is_cfi = false
 
   is_component_build = false
-  symbol_level = 1
-  treat_warnings_as_errors = true
+
+  treat_warnings_as_errors = false
```

Comment:
unrelated / bad merge. also you seem to be updating your repo via pull requests that leave merge commits - you should be fast-forwarding.